packages/angular/projects/clr-angular/src/popover/tooltip/_tooltips.clarity.scss

```diff
@@ -215,8 +215,11 @@
       margin-right: 0;
     }
 
-    cds-icon > svg {
-      pointer-events: none;
+    cds-icon,
+    clr-icon {
+      svg {
```

Comment:
This should probably be `& > svg` to better reflect the original SASS. I don't think it makes that much difference. But just calling out that...

```
cds-icon { 
    svg {
```

...is not the same as....

`cds-icon > svg {`

Reply:
Thank you for catching that!